ipushp/views/website_ipushp_template.xml

```diff
@@ -353,47 +358,23 @@
         <t t-call="website.layout">
             <div id="wrap">
                 <div class="oe_structure" />
+                <link rel="stylesheet"
+                    href="https://2-22-4-dot-lead-pages.appspot.com/static/lp918/min/default_thank_you.css" />
```

Comment:
@nirmal-serpentcs Instead of calling the external CSS. load those files in the module.